hasher-matcher-actioner/terraform/fetcher/main.tf

```diff
@@ -81,13 +83,12 @@ resource "aws_iam_role" "fetcher" {
 data "aws_iam_policy_document" "fetcher" {
   statement {
     effect    = "Allow"
-    actions   = ["s3:GetObject"]
-    resources = ["arn:aws:s3:::${var.threat_exchange_data.bucket_name}/${var.threat_exchange_data.pdq_data_file_key}"]
-  }
-  statement {
-    effect    = "Allow"
-    actions   = ["s3:PutObject"]
-    resources = ["arn:aws:s3:::${var.threat_exchange_data.bucket_name}/${var.threat_exchange_data.pdq_data_file_key}"]
+    actions   = [
+      "s3:GetObject",
+      "s3:PutObject",
+      "s3:ListBucket",
+    ]
+    resources = ["*"]
```

Comment:
I'm not an expert, but is `resources = ["*"]` an OK practice?

Reply:
True, this could be scoped more narrowly.

Reply:
fwiw I searched the existing terraform scripts and only found `resources = ["*"]` in things relating to logging in cloudfront.

Reply:
I tried a few different ways but the most specific I could get without failing seems to be to limit to s3. I've added this